views/addmarker.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset=utf-8 />
+<title>Add marker</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+<style>
```

Comment:
@sos0 @waazup2: Are you two okay with the inline CSS here, or should we move it?